idl-compiler.py

```diff
@@ -74,6 +74,30 @@ def ns_qualified_name(self):
         return self.name if not self.ns_context \
             else self.combine_ns(self.ns_context) + "::" + self.name
 
+class Import(ASTBase):
+    '''AST node representing a single `import module`.'''
+
+    def __init__(self, name):
+        super().__init__(name)
+
+    def __str__(self):
+        return f"<Import(name={self.name})>"
+
+    def __repr__(self):
+        return self.__str__()
+
+class Include(ASTBase):
```

Comment:
What is the difference between `import` and `include`?

Reply:
I see it down below, but we need to document it somewhere. Speaking of which, this whole IDL is woefully under-documented (or documentation is just very hard to discover). Not this series's fault of course.

Reply:
`include` just adds a respective `#include` directive in the generated `.dist.hh` header using the same parameter.

`import name` refers to another `name.idl.hh` "module" and adds `#include "name.dist.hh"` and  `#include "name.dist.impl.hh"` to the respective generated header files.